### PR TITLE
Feat/remove all of tina (Do not merge until 2-3 approvals )

### LINKED ISSUE #1216: Remove tina package from `_app.js`

### PR DESCRIPTION
**Pull request by logan-anderson**
This pr removes all the tina packages from this site ~~except for `@tinacms/styles` (maybe @spbyrne knows how to remove this).~~ (Thanks @spbyrne  for fixing this)

This PR requires a lot of review. It is a pretty big change that could break some things so I would love if I could get 2-3 approvals on this before it is merged.

closes #1216

### CODE REVIEW

**Comment by spbyrne** [comment]
I'll take a look at the `@tinacms/styles` stuff.

**Comment by spbyrne** [comment]
I've incorporated the css vars we're using from the styles package and removed it. I don't think we'll use CSS vars like that with new Tina ui going forward so it's a bit of a vestige at this point.

**Comment by spbyrne** [comment]
Are `@tinacms/webpack-helpers` and `@tinacms/graphql` still needed?

**Comment by logan-anderson** [comment]
@spbyrne 
> Are @tinacms/webpack-helpers and @tinacms/graphql still needed?

I removed `@tinacms/webpack-helpers` and we are using `@tinacms/graphql` for some docs stuff so it will have to stay in.

File: utils/shortcodes.tsx [hunk 1]
```diff
@@ -17,26 +17,27 @@ const DemoButtonWrapper = styled.div`
 `
 
 export function AlertTest({ type, message, timeout, buttonText }) {
-  const cms = useCMS()
-  const sendAlert = React.useCallback(() => {
-    cms.alerts[type](message, timeout)
-  }, [type, message, timeout])
+  // TODO: investigate what removing this broke and if it needs to be added in
```
Comment (logan-anderson) [inline utils/shortcodes.tsx:20]: i am looking into this now

**Review by spbyrne — DISMISSED** [review]
Looks good on my end. Agreed more approvals should be required for a merge, though.

**Review by jamespohalloran — APPROVED** [review]
Poked through the site preview, and all looks good to me!

**Review by ashleyjanedoiron — APPROVED** [review]
LGTM!

**Review by jamespohalloran — APPROVED** [review]

**Pull request merged**